djautotask/api.py

```diff
@@ -192,7 +192,7 @@ def update_object(at_object, status):
     # to autotask unless we sync EVERY non-readonly field. If you submit
     # the object with no values supplied for the readonly fields,
     # autotask will null them out.
-    entity = at_object.type_name.capitalize()
+    entity = at_object.__class__.__name__
```

Comment:
Didn't realize this on the first look. I think we'll have to go back to using `type_name` cause this resolves to `AutotaskTicketCard` instead of `Ticket` and that throws an error when trying to change ticket status.

```python
[2020-02-27 14:19:05] DEBUG [kanban.board.views.card_detail:112] Changing AT card 8671 status from
 Denied to Approved.                                                                              
> /home/cameron/Dev/kanban-dev/env3/lib/python3.6/site-packages/django_autotask-0.0.55a0-py3.6.egg
/djautotask/api.py(196)update_object()                                                            
-> entity = at_object.__class__.__name__                                                          
(Pdb) at_object                                                                                   
<AutotaskTicketCard: AT card 8671>                                                                
(Pdb) at_object.__class__.__name__                                                                
'AutotaskTicketCard'                                                                              
```